packages/beacon-node/test/unit/chain/validation/attestation/getShufflingForAttestationVerification.test.ts

```diff
@@ -45,7 +45,7 @@ describe("getShufflingForAttestationVerification", () => {
         throw new Error("Unexpected input");
       }
     });
-    const expectedShuffling = {epoch: attEpoch} as EpochShuffling;
+    const expectedShuffling = {epoch: attEpoch} as unknown as EpochShuffling;
```

Comment:
`epoch` is no longer on EpochShuffling but using it here so there is something on the object to assert that its the same object